perfkitbenchmarker/benchmarks/silo_benchmark.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs Silo.
+
+Silo is a high performance, scalable in-memory database for modern multicore
+machines
+
+Documentation & code: https://github.com/stephentu/silo
+"""
+
+import logging
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import regex_util
+from perfkitbenchmarker import sample
+from perfkitbenchmarker.packages import silo
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_INFO = {'name': 'silo',
+                  'description': 'Runs Silo',
+                  'scratch_disk': False,
+                  'num_machines': 1}
+
+flags.DEFINE_string('silo_benchmark', 'tpcc', 'benchmark to run with silo')
```

Comment:
Might want to give what input is possible here in addition to tpcc like this:
The benchmark to run with SILO.  Options include tpcc, ...